static.js

```diff
@@ -65,6 +65,8 @@ module.exports = function (fastify, opts, next) {
     }
 
     fastify.register(require('./routes'), options)
+  } else {
+    console.log()
```

Comment:
why this empty `console.log` was added here?

Reply:
Otherwise, the coverage tool says that `else path not taken` and treats it as an uncovered branch. I didn't know how to do it the other way. That's why I inserted an empty `console.log`. Though, that's not the best option. 

I saw some tests in `fastify` repo, you don't have such a problem. Do you ignore such messages from the coverage somehow?